### PR TITLE
Fix for #12 + Name Change

### PR DESCRIPTION
**Pull request by TurboProgramming**
This PR includes a fix for issue #12 and a small name change from Jorge to Jorge VB upon request from community (https://community.makeroid.io/t/testimonials-for-the-makeroid-website/4853/12)

### CODE REVIEW

**Comment by TurboProgramming** [comment]
I need to do some last changes on this before it gets merged. Please wait until I complete them. @conorshipp

**Comment by TurboProgramming** [comment]
@conorshipp Completed all.

File: _data/testimonials.yml [hunk 1]
```diff
@@ -12,19 +12,19 @@
...
+  text: The first time I tried Makeroid, I was impressed with the number of components that it got. It’s also very easy to learn, even for the most beginners.
 
 - name: Kanishka Chakraborty
   username: Kanishka_Developer
   text: Makeroid is hands down the fastest way to make an app, and lets me make production-ready apps directly on my Android tablet.
 
 - name: Domi Rami
   username: Domi
-  text: With Makeroid I find it easier to make my apps for the number of options offered to the development, all very intuitive and you get help from the community.
+  text: With Makeroid its easier to make my apps for the number of options offered to the development, all very intuitive and you get help from the community.
```
Comment (conorshipp) [inline _data/testimonials.yml]: Should be `it's` rather than `its`.

File: _data/testimonials.yml [hunk 2]
```diff
@@ -12,19 +12,19 @@
...
 - name: Carlos Pedroza
   username: carlospedroza27
-  text: Makeroid is merging the ability to create awesome android applications without writing a single line of code and the ability to make a professional app in a stunning way.
+  text: Makeroid is merging the ability to create awesome Android apps without writing a single line of code and the ability to make an app in a stunning way.
 
-- name: Jorge
+- name: Jorge VB
   username: J.V.B
-  text: The first time I tried Makeroid, I was really impressed with the number of components that Makeroid offered me. It’s much more flexible than others, and there’s always a community waiting to hear your problems, ideas, or suggestions. It’s also very easy to learn, even for the most beginner.
+  text: The first time I tried Makeroid, I was impressed with the number of components that it got. It’s also very easy to learn, even for the most beginners.
```
Comment (conorshipp) [inline _data/testimonials.yml]: A couple of parts here don't make much sense. Can you change `it got` to `it has got`? Can you remove `the` from `even for the most beginners`?

File: _data/testimonials.yml (hunk 1 above)
Comment (TurboProgramming) [reply _data/testimonials.yml]: Fixed.

File: _data/testimonials.yml (hunk 2 above)
Comment (TurboProgramming) [reply _data/testimonials.yml]: Fixed @conorshipp.

File: _data/testimonials.yml [hunk 3]
```diff
@@ -12,19 +12,19 @@
...
 - name: Carlos Pedroza
   username: carlospedroza27
-  text: Makeroid is merging the ability to create awesome android applications without writing a single line of code and the ability to make a professional app in a stunning way.
+  text: Makeroid is merging the ability to create awesome Android apps without writing a single line of code and the ability to make an app in a stunning way.
 
-- name: Jorge
+- name: Jorge VB
   username: J.V.B
-  text: The first time I tried Makeroid, I was really impressed with the number of components that Makeroid offered me. It’s much more flexible than others, and there’s always a community waiting to hear your problems, ideas, or suggestions. It’s also very easy to learn, even for the most beginner.
+  text: The first time I tried Makeroid, I was impressed with the number of components that it has got. It’s also very easy to learn, even for most beginners.
```
Comment (conorshipp) [inline _data/testimonials.yml]: Actually, after reading this again, it's still not right. It should be `...components that it had`.

Comment (TurboProgramming) [reply _data/testimonials.yml]: Completed.

**Pull request merged**